Add a db interact command

ttung opened the pull request:
### Description
This allows devs to interact with the database via ipython.

### Test plan
```
% aspen-cli db interact
Python 3.7.6 (default, Dec 22 2019, 01:09:06)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.20.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: session = engine.make_session()

In [2]: session.query(User).all()
Out[2]: []

In [3]:
```

phoenixAja reviewed (approved):
would be nice to have instructions for this in the docs (which could just be the screenshot you added)

ttung commented:
added docs.

Commit:
Add a db interact command

### Description
This allows devs to interact with the database via ipython.

### Test plan
```
% aspen-cli db interact
Python 3.7.6 (default, Dec 22 2019, 01:09:06)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.20.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: session = engine.make_session()

In [2]: session.query(User).all()
Out[2]: []

In [3]:
```
